test_unstructured/staging/test_weaviate.py

```diff
@@ -53,6 +54,7 @@ def test_stage_for_weaviate(filename="example-docs/layout-parser-paper-fast.pdf"
     }
 
 
+@pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
```

Comment:
I know there are a lot like this, but this reason is not a reason, it just restates the "skip_outside_ci" label. A reason would answer the question "Why do we skip this when we're not running in the CI environment?"